src/server/routers/init.js

```diff
@@ -1,11 +1,11 @@
 const express = require('express');
 
 const {
-    addBookToDB
+    seedBookDatabase
 } = require('../controllers/init.js');
 
 const router = express.Router();
 
-router.post('/', addBookToDB);
+router.post('/', seedBookDatabase);
```

Comment:
Looks good!

Reply:
Ooooo - I reckon this might mean '/' resolves correctly on heroku, right?

Reply:
I didn't take a look at the heroku thing yet. I'll take a look now and come back to you.